08-projects.Rmd

```diff
@@ -5374,7 +5399,7 @@ The next couple of questions should be complete in the same language. You can us
 
 ##### 4. Now that you have a working query, connect to the database and run the query to get the data into an R or `pandas` data frame. In previous projects, we learned how to used regular expressions to search for text. For each character, how many episodes `primary_title`s contained their name? 
 
-**Relevant topics:** [SQL in R](#sql-in-r), [SQL in Python](#sql-in-python), [grep](#r-grep)
+**Relevant topics:** [SQL in R](#r-sql), [SQL in Python](#sql-in-python), [grep](#r-grep)
```

Comment:
Hi @kevinamstutz , while I changed the link, it seems that `#r-sql` is still an empty section in the Examples Book. Do you want to link the "SQL in R" section under the SQL chapter (https://thedatamine.github.io/the-examples-book/sql.html)?

Reply:
I think I'm just going to go ahead and add a little bit of content to it so it's not still empty! Thanks!

Reply:
Thanks for editing the book!